elastic/detection-rules
# [Docs | Rule Tuning] Add blog references to rules

# Pull Request

*Issue link(s)*: N/A



## Summary - What I changed



- This PR adds our blog content as references to our prebuilt rules. 
- Noticed some auto formatting was added on toml_save.

## How To Test



- Check out any of the blogs added as a reference and double check that the rule is mentioned in the blog.

## Checklist
